cryptography/src/bls12381/dkg/player.rs

```diff
@@ -145,28 +141,18 @@ impl<P: PublicKey, V: Variant> Player<P, V> {
         for (idx, share) in reveals {
```

Comment:
Should we not be checking all commitments provided to finalize?

We only seem to verify commitments for reveals (don't seem to check in VRF either: https://github.com/commonwarexyz/monorepo/blob/b8610fc03589df33f9913c4e6ad2f376971807f4/examples/vrf/src/handlers/contributor.rs#L417-L430) 🤔

Reply:
It is a bit confusing because we use the commitments from `self.dealings` and don't compare them with what is provided for a given dealer?

Reply:
We should add a test for dealer equivocation (and ensure we can update our tracked commitment/share).